pkg/api/convertertoadmin.go

```diff
@@ -104,6 +104,7 @@ func ConvertToAdmin(cs *OpenShiftManagedCluster) *admin.OpenShiftManagedCluster
 func convertConfigToAdmin(cs *Config) *admin.Config {
 	return &admin.Config{
 		ClusterVersion:                       &cs.ClusterVersion,
+		ClusterLogLevel:                      convertClusterLogLevelsToAdmin(cs.ClusterLogLevel),
```

Comment:
there are no changes made to convertfromadmin.go and there should be.  This should have triggered a unit test failure.  Add a unit test that does a round trip: populate a dummy `*admin.OpenShiftManagedCluster`, convert it to a `*OpenShiftManagedCluster`, convert that back again and check that what you get equals what you started with.

Reply:
thats right, theres even more stuff that had been left out in conversions

```
=== RUN   TestRoundTrip
--- FAIL: TestRoundTrip (0.00s)
    converterfromadmin_test.go:301: unexpected diff [Config.ClusterVersion: Config.ClusterVersion !=  Config.ClusterLogLevel.ApiServer: 1 != 0 Config.ClusterLogLevel.ControllerManager: 1 != 0 Config.ClusterLogLevel.Node: 1 != 0 Config.GenevaLoggingControlPlaneEnvironment: Config.GenevaLoggingControlPlaneEnvironment !=  Config.GenevaLoggingControlPlaneRegion: Config.GenevaLoggingControlPlaneRegion != ]
```

I'll add them in